lib_test/.merlin

```diff
@@ -20,3 +20,4 @@ S bap_sema
 S bap_types
 S bap_trace  
 PKG oUnit
+PKG pcre
```

Comment:
I think that it should be removed... although it is minor, make sure that you remove it in the next PR.